build/build.umple.xml

```diff
@@ -40,6 +41,12 @@
     <delete dir="cruise.umple.validator/src-gen-umple" failonerror="false" />
   </target>
 
+  <target name="init">
+    <antcall target="template.init" />
+    <!-- TODO: get the "-f build.deps.xml bootstrap" to be part of the docs / build and then remove this line -->
+    <ant antfile="build/build.deps.xml" target="deps.vendor" inheritAll="true" />

```

Comment:
Grabbing vendors doesn't take long, but I would like to make the "bootstrap" more streamlined into our "first-build", but for now I went with the safer always update your vendor dependencies on build